### PR TITLE
Drop gulp and babel.

### PR DESCRIPTION
**Pull request by XhmikosR**
This is WIP. The plan is to simplify things.

I haven't tested any deployment yet.

### CODE REVIEW

**Comment by XhmikosR** [comment]
@emedvedev what do you think about this? I think it's a great improvement given we are getting rid of so many packages and we simplify things. And since we have CI, we should be covered.

File: package.json [hunk 1]
```diff
@@ -29,53 +26,38 @@
     "pug": "^2.0.4",
     "socket.io": "^2.3.0",
     "superagent": "^3.8.3",
-    "tinycolor2": "^1.4.1"
+    "tinycolor2": "^1.4.1",
+    "node-sass": "^4.12.0"
```
Comment (XhmikosR) [inline package.json]: I moved node-sass to deps on purpose so that we don't need extra stuff on Heroku.

File: package.json [hunk 2]
```diff
@@ -29,53 +26,38 @@
...
     "lint": "eslint bin/slackin lib test",
-    "mocha": "mocha --require @babel/register --require ./test/setup --exit",
-    "test": "npm run lint && npm run mocha",
-    "build": "gulp",
+    "mocha": "mocha --require ./test/setup --exit",
+    "test": "npm run build && npm run lint && npm run mocha",
+    "build": "node-sass --output-style compressed --precision 6 scss/ -o assets/",
     "start": "./bin/slackin",
-    "heroku-prebuild": "npm install --only=dev",
-    "heroku-postbuild": "gulp"
```
Comment (XhmikosR) [inline package.json]: I removed these scripts since AFAICT Heroku does run the build command automatically.

File: package.json [hunk 3]
```diff
@@ -29,53 +26,38 @@
...
+    "prebublish": "npm run build"
```
Comment (XhmikosR) [inline package.json]: I added this to make sure you don't miss to generate the CSS before deployments.

**Comment by emedvedev** [comment]
I think it's a good idea! And it should work reasonably well with the CI changes. I'll take a closer look at how the end-user workflow would change (if at all), and if there's no extra complication for the users, I'd be more than happy to merge it.

**Comment by XhmikosR** [comment]
Glad you like it :)

Let's finish the other PR, and I'll rebase this so that you test a deployment. I still can't test on Heroku due to #20

**Comment by XhmikosR** [comment]
I think we can move with this. I tested it, and works (as far as our deployment can go since we are hitting #20 ). Just remember we need to make this a major version bump

**Comment by emedvedev** [comment]
Alright, I've been trying it, and it looks fine, so we can merge. I'll bump the version accordingly, although it's probably better to merge all the PRs and make all the changes first.

It's still marked as a draft, is there anything else you want to accomplish in this PR?

**Comment by XhmikosR** [comment]
I was actually thinking we should take the chance and update to Node.js 10. But this should be done in a separate PR and I'll probably need some help because there are many deployment methods :)

**Comment by XhmikosR** [comment]
Alright, I pushed another minor patch and rebased this. I tested it locally and works.

**Comment by XhmikosR** [comment]
I updated the readme too. Please confirm

**Comment by emedvedev** [comment]
Let's go! And thank you soooo much for all your work on this project, it's much appreciated, really.

**Pull request merged**